### PR TITLE
[Do no merge][test] feat: add TCP integration

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

test to create a TCP input integration.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/tcp/0.0.1/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+format_version: 1.0.0
```
Comment (mtojek) [inline packages/tcp/0.0.1/manifest.yml:1]: You have to remove the version from directories.

**Comment by kuisathaverat** [comment]
I have to debug this error when I add the integration

Screenshot 2021-05-20 at 12 24 12

**Comment by kuisathaverat** [comment]
No Kibana errors in logs

```
{
  "type": "response",
  "@timestamp": "2021-05-20T10:25:42+00:00",
  "tags": ["access:fleet-read"],
  "pid": 980,
  "method": "get",
  "statusCode": 200,
  "req": {
    "url": "/api/fleet/epm/packages/tcp-0.0.1",
    "method": "get",
    "headers": {
      "host": "127.0.0.1:5601",
      "connection": "keep-alive",
      "sec-ch-ua": "\" Not A;Brand\";v=\"99\", \"Chromium\";v=\"90\", \"Google Chrome\";v=\"90\"",
      "dnt": "1",
      "sec-ch-ua-mobile": "?0",
      "user-agent": "FOO",
      "kbn-version": "7.13.0-SNAPSHOT",
      "content-type": "application/json",
      "accept": "*/*",
      "sec-fetch-site": "same-origin",
      "sec-fetch-mode": "cors",
      "sec-fetch-dest": "empty",
      "referer": "http://127.0.0.1:5601/app/fleet",
      "accept-encoding": "gzip, deflate, br",
      "accept-language": "en,en-US;q=0.9,en-GB;q=0.8"
    },
    "remoteAddress": "192.168.1.18",
    "userAgent": "FOO",
    "referer": "http://127.0.0.1:5601/app/fleet"
  },
  "res": {
    "statusCode": 200,
    "responseTime": 41,
    "contentLength": 2250
  },
  "message": "GET /api/fleet/epm/packages/tcp-0.0.1 200 41ms - 2.2KB"
}
```

**Comment by kuisathaverat** [comment]
This is the response from Kibana
```
{
  "response": {
    "name": "tcp",
    "title": "TCP input",
    "version": "0.0.1",
    "release": "experimental",
    "description": "TCP input to read events over TCP.\n",
    "type": "integration",
    "download": "/epr/tcp/tcp-0.0.1.zip",
    "path": "/package/tcp/0.0.1",
    "icons": [{
      "src": "/img/icon.svg",
      "path": "/package/tcp/0.0.1/img/icon.svg",
      "type": "image/svg+xml"
    }],
    "format_version": "1.0.0",
    "readme": "/package/tcp/0.0.1/docs/README.md",
    "license": "basic",
    "categories": ["custom"],
    "assets": {},
    "policy_templates": [{
      "name": "tcp",
      "title": "TCP input",
      "description": "TCP input to read events over TCP",
      "inputs": [{
        "type": "tcp",
        "title": "TCP input",
        "description": "TCP input to read events over TCP."
      }],
      "multiple": true
    }],
    "data_streams": [{
      "type": "logs",
      "dataset": "tcp.tcp",
      "title": "TCP Dataset",
      "release": "experimental",
      "streams": [{
        "input": "TCP",
        "vars": [{
          "name": "max_message_size",
          "type": "text",
          "title": "Maximum size of the message",
          "description": "The maximum size of the message received over TCP. The default is 20MiB.",
          "multi": false,
          "required": false,
          "show_user": false,
          "default": "20MiB"
        }, {
          "name": "host",
          "type": "text",
          "title": "The host and TCP port to listen",
          "description": "The host and TCP port to listen on for event streams.",
          "multi": false,
          "required": true,
          "show_user": false,
          "default": "0.0.0.0:9000"
        }, {
          "name": "data_stream.dataset",
          "type": "text",
          "title": "Dataset name",
          "description": "Set the name for your dataset. Changing the dataset will send the data to a different index. You can't use `-` in the name of a dataset and only valid characters for [Elasticsearch index names](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html).\n",
          "multi": false,
          "required": true,
          "show_user": false,
          "default": "generic"
        }, {
          "name": "custom",
          "type": "yaml",
          "title": "Custom configurations",
          "description": "Here YAML configuration options can be used to be added to your configuration. Be careful using this as it might break your configuration file.\n",
          "multi": false,
          "required": false,
          "show_user": false,
          "default": ""
        }],
        "template_path": "stream.yml.hbs",
        "title": "Collect TCP events",
        "description": "TCP input to read events over TCP.",
        "enabled": true
      }],
      "package": "tcp",
      "path": "tcp"
    }],
    "owner": {
      "github": "elastic/integrations-services"
    },
    "latestVersion": "0.0.1",
    "removable": true,
    "status": "not_installed"
  }
}
```

**Pull request closed without merging**